4.-Console-app-calls-web-API-with-PoP/TodoListClient/MainWindow.xaml.cs

```diff
@@ -286,9 +224,12 @@ private async void AddTodoItem(object sender, RoutedEventArgs e)
                 return;
             }
 
+            //
             // Call the To Do service.
+            //
+
             // Once the token has been returned by MSAL, add it to the http authorization header, before making the call to access the To Do service.
-            _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", result.AccessToken);
+            _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("PoP", result.AccessToken);
```

Comment:
This is different with Pop.
we pass the request, and therefore this is not needed.

Do we really want to replace the console app by the WPF app?
this is way more complicated for Pop?

cc: @kalyankrishna1 what do you think?

Reply:
Changed the client application to console as suggested.
Thank you.